yang/ietf-l3vpn-ntw.yang

```diff
@@ -2125,6 +2141,84 @@ module ietf-l3vpn-ntw {
                           "Indicates whether IPv4, IPv6, or both
                            address families are to be activated.";
                       }
+                      container timers {
+                        description
+                          "Indicates the RIP timers.";
+                        reference
+                         "RFC 8695: A YANG Data Model for the 
+                         Routing Information Protocol (RIP)";
+                        leaf update-interval {
+                           type uint16 {
+                             range "1..32767";
+                           }
+                           units "seconds";
+                           default "30";
+                          description
+                            "Indicates the RIP update time.
+                             The amount of time for which 
+                             routing updates are sent.";
+                        }
+                        leaf invalid-interval  {
+                          type uint16 {
+                             range "1..32767";
+                           }
+                           units "seconds";
+                           default "180";
+                          description
+                            "Interval before a route is declared 
+                            invalid after no updates are received. 
+                            This value is at least three times
+                            the value for the update-interval argument.";
+                        }
+                        leaf holddown-interval {
+                           type uint16 {
+                             range "1..32767";
+                           }
+                           units "seconds";
+                           default "180";
+                          description
+                            "Interval before better routes are released.";
+                        }
+                        leaf flush-interval {
+                            type uint16 {
+                               range "1..32767";
+                             }
+                             units "seconds";
+                             default "180";
+                          description
+                            "Indicates the RIP  flush timer
+                             The amount of time (in seconds) 
+                             that must pass before a route 
+                             is removed from the routing table.";
+                        }
+                      }
+                      leaf-list neighbor {
+                        type inet:ip-address;
+                        description
+                          "Indicates the IP address of the peer.";
+                      }
+                      uses vpn-common:service-status;
+                    }
```

Comment:
You may include: 

    leaf default-metric {
      type uint8 {
        range "0..16";
      }
      default "1";
      description
        "Set the default metric.";
    }

Reply:
agree! please see 2c4a66fcade6e49cab82803ef4ce72d229f54c16